docs/schema/index-management/index-lifecycle.md

```diff
@@ -26,28 +26,72 @@ mgmt.commit();
 
 ## Index states and transitions
 
-![States and transitions](index-lifecycle.png)
+The diagram below shows the possible states of an index in JanusGraph.
+Transitions which are not recommended or should be used with caution are shown in grey.
+
+![States and transitions](index-lifecycle.svg){: style="width:100%;"}
 
 ## States (SchemaStatus)
 An index can be in one of the following states:
 
--   **INSTALLED** The index is installed in the system but not yet registered with all instances in the cluster
+**INSTALLED**
 
--   **REGISTERED** The index is registered with all instances in the cluster but not (yet) enabled
+:   The index has been created on the current JanusGraph instance, but it is not yet known to all instances in the cluster.
+    This state is not shown in the diagram above because it is not stable in terms of synchronization between instances.
 
--   **ENABLED** The index is enabled and in use
+**REGISTERED**
 
--   **DISABLED** The index is disabled and no longer in use
+:   The index is known to all instances in the cluster.
+    However, it is not in use yet.
+    In this state, the index contains no data and is not eligible to be used in queries.
 
-## Actions (SchemaAction)
-The following actions can be performed on an index to change its state via `mgmt.updateIndex()`:
+**ENABLED**
+
+:   The index is enabled and can be used to answer queries.
+    Furthermore, this is the only state in which insertions and deletions in the graph are forwarded to the index.
 
--   **REGISTER_INDEX** Registers the index with all instances in the graph cluster. After an index is installed, it must be registered with all graph instances
+**DISABLED**
 
--   **REINDEX** Re-builds the index from the graph
+:   The index is disabled and will not be used to answer queries.
+It also does no longer receive graph updates.
+Therefore, it is recommended to use `REINDEX` to re-enable the index.
 
--   **ENABLE_INDEX** Enables the index so that it can be used by the query processing engine. An index must be registered before it can be enabled.
+**DISCARDED**
 
--   **DISABLE_INDEX** Disables the index in the graph so that it is no longer used.
+:   The index has been decommissioned and its contents have been deleted.
+    The only remaining artifact is the schema vertex representing the index.
+
+## Actions (SchemaAction)
+The following actions can be performed on an index to change its state via `mgmt.updateIndex()`:
 
--   **REMOVE_INDEX** Removes the index from the graph (optional operation). Only on composite index.
+**REGISTER_INDEX**
+:   Registers the index with all instances in the graph cluster.
```

Comment:
I think graph cluster is a confusing term. The next sentence even speaks of _graph instances_ which should be _JanusGraph instances_.
Not sure what to do with _graph cluster_. Maybe just drop the _graph_?